tools/build_file_generator/BUILD

```diff
@@ -129,73 +114,3 @@ checkstyle_test(
     name = "build_file_generator_tool-checkstyle",
     target = ":build_file_generator_tool",
 )
-
```

Comment:
To run this in other repos (e.g `hasadna`), you can still use the same code (ThirdPartyDepsTool), it's just the BUILD file that needs to be define in `hasadna`, no?

Or do you mean this would allow `hasadna` to reference code in `startup-os` more easily?